webdriver-spec.html

```diff
@@ -2843,7 +2843,7 @@
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+ <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
```

Comment:
Yes, but if the current browsing context is an `<iframe>` and this element has been removed from the DOM, the top-level browsing context still exists. Since the Back command is one of those commands that operate independently of frames, it should still be possible to call it even if the frame one has switched into has gone missing.